hpobench/dependencies/ml/data_manager.py

```diff
@@ -0,0 +1,199 @@
+import openml
+import numpy as np
+import pandas as pd
+from typing import Union
+from pathlib import Path
+
+from sklearn.impute import SimpleImputer
+from sklearn.pipeline import make_pipeline
+from sklearn.utils import check_random_state
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.preprocessing import StandardScaler
+from sklearn.model_selection import train_test_split
+from oslo_concurrency import lockutils
+
+from hpobench.util.data_manager import DataManager
+
+
+from hpobench import config_file
+
+
+class OpenMLDataManager(DataManager):
```

Comment:
I am not testing this...trust you with this interface ;) 
but looks good, maybe later while building the README, we can include an example of its use?